products/apache-activemq.md

```diff
@@ -47,7 +47,7 @@ releases:
 
 -   releaseCycle: "5.17"
     releaseDate: 2022-03-09
-    eol: false # still listed on https://activemq.apache.org/components/classic/download/
+    eol: 2024-04-12
```

Comment:
where can I see these dates?

Reply:
Based on the history for this page:

https://github.com/apache/activemq-website/commits/main/src/_data/current_releases.yml

Specifically,

- [This commit](https://github.com/apache/activemq-website/commit/021a32970405d98b52f647cb838e7a91f4c2b5dc) drops 5.17 on 12th April
- [This commit](https://github.com/apache/activemq-website/commit/8ae93ea694313f816646ba5adf2e94dd1c665ef6) drops 6.0 on 17th March.

The YAML file is the source for what is rendered at https://activemq.apache.org/components/classic/download/, which we already link to, so we're just tracking history of the page, in a way.